sql-plugin/src/main/330+/scala/org/apache/spark/sql/rapids/shims/RapidsErrorUtilsFor330plus.scala

```diff
@@ -26,6 +26,10 @@ trait RapidsErrorUtilsFor330plus {
     QueryExecutionErrors.divideByZeroError(origin.context)
   }
 
+  def intervalDivByZeroError(origin: Origin): ArithmeticException = {
+    divByZeroError(origin)
```

Comment:
Should this be
```suggestion
    QueryExecutionErrors.divByZeroError(origin)
```

Reply:
No, because for version prior to 340, dividing an interval by zero still throws a divideByZeroError.